bin/ttmscalr

```diff
@@ -1201,4 +1202,33 @@ Main {
       url_shard(replacement)
     end
   end
+
+  mode 'psql:tunnel' do
+    description 'Open an SSH tunnel to an RDS DB'
```

Comment:
No, it actually opens the tunnel to the DB Server for the farm, it just routes it through the debug role. If you look in ssher.rb line 32 you can see where I build the tunnel spec. The only thing available on the tunnel is the Postgres server. Ill update the wording a bit though.